<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericFunctionDialog</class>
 <widget class="QDialog" name="GenericFunctionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Function Selection</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>270</width>
     <height>145</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="doubleDoubleButton">
          <property name="text">
           <string>doubleDouble</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="doubleDoubleVectorButton">
          <property name="text">
           <string>doubleDoubleVector</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="reverseStringButton">
          <property name="text">
           <string>reverseString</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="squareIntButton">
          <property name="text">
           <string>squareInt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="squareIntVectorButton">
          <property name="text">
           <string>squareIntVector</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>25</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
